src/mainwindow.cpp

```diff
@@ -209,7 +209,7 @@ void MainWindow::ResetView()
 
 void MainWindow::OpenVideoFile()
 {
-	QString file = QFileDialog::getOpenFileName(this, tr("Open Video"));
+        QString file = QFileDialog::getOpenFileName(this, tr("Open Video"), QDir::currentPath(), tr("All files (*)"), Q_NULLPTR, QFileFialog::DontUseNativeDialog);
```

Comment:
```suggestion
	QString file = QFileDialog::getOpenFileName(this, tr("Open Video"), QDir::currentPath(), tr("All files (*)"), Q_NULLPTR, QFileFialog::DontUseNativeDialog);
```

Reply:
Please keep the tab

Reply:
Mmm sorry... emacs chage it without asking

Reply:
use vim ;-P